contrib/ci-horizen/scripts/common/install.sh

```diff
@@ -25,7 +25,7 @@ if [ "${TRAVIS_OS_NAME}" = "linux" ]; then
 fi
 
 if [ "${TRAVIS_OS_NAME}" = "osx" ]; then
-  brew install ${UPDATE_PACKAGES}
+  #brew install ${UPDATE_PACKAGES}
```

Comment:
Better to test if var is not empty, we might want to install other brew dependencies in the future.
```suggestion
if [ ! -z "${UPDATE_PACKAGES}" ]; then
  brew install ${UPDATE_PACKAGES}
fi
```